Readme.md

```diff
@@ -2,31 +2,106 @@
 
```

Comment:
You mean the hard wraps and some language/english rephrasing ?

I made them part of:

```
commit 3875bc8
Author: Rafael David Tinoco <rafaeldtinoco@gmail.com>
Date:   Thu Jul 8 13:19:29 2021

    Update Readme.md with build instructions and minor fixes
````

But I can either revert them if you dont think appropriate

> OR split them into 2 commits. If the tree was older I would probably have done 2 commits but I thought it was ok doing all at once since we're in the beginning (at least until the initial releases from this tree).

Reply:
Oh, I just realized there is a pattern in libbpfgo and tracee for no wraps, I'll revert the wraps and keep the build instructions and some better rephrasing if that is ok for you.

Reply:
Fixed =). 2 commits. 1 just for small rephrasing. No format changes (same as other .md files). Build instructions added.